api/commands/query.ts

```diff
@@ -58,6 +58,10 @@ export async function compile(
   // Resolve the path in case it hasn't been resolved already.
   const projectDir = path.resolve(compileConfig.projectDir);
 
+  // This captures any single backticks that aren't escaped with a preceding \.
+  const RAW_BACKTICKS_REGEX = /(?!\\)`/g;
```

Comment:
i'm not clear on why we only want to do this for backticks that aren't already escaped; IMO that's confusing. i think we should do it for *all* backticks; if you're escaping one then you want to send those bytes (i.e. a backslash and a backtick) to your warehouse; if we don't re-escape the backtick then those bytes will *not* be sent to the warehouse.

Reply:
Ah yes, that makes sense